apps/google-analytics-4/frontend/src/hooks/useSidebarSlug/useSidebarSlug.tsx

```diff
@@ -33,7 +33,9 @@ export const useSidebarSlug = (slugFieldInfo: ContentTypeValue) => {
     sdk.entry.onSysChanged((sys) => handlePublishedStatus(sys));
   }, [sdk.entry]);
 
-  const reportSlug = `/${pathJoin(urlPrefix || '', debouncedSlugFieldValue || '')}`;
+  const reportSlug = `/${pathJoin(urlPrefix || '', debouncedSlugFieldValue || '')}${
+    useTrailingSlash ? '/' : ''
+  }`;
```

Comment:
Would this create a double / if the field value already ends in a /?

If so, you can take the regex from here:
https://github.com/contentful/marketplace-partner-apps/blob/main/apps/shopify/src/dataTransformer.js#L22

and do something like:
```js
const path = pathJoin(urlPrefix || '', debouncedSlugFieldValue || '';

const trailingSlash = /\/$/;
const reportSlug = useTrailingSlash ? path.replace(trailingSlash, '') + '/' : path;
```

Reply:
Good callout! I confirmed that this line takes care of it already for us: https://github.com/contentful/apps/blob/master/apps/google-analytics-4/frontend/src/utils/pathJoin.ts#L3 and have verified that a slug name with a trailing slash will not have a second one added on.